homeassistant/components/sensor/efergy.py

```diff
@@ -30,8 +30,8 @@
 CONF_COST = 'cost'
 
 SENSOR_TYPES = {
-    CONF_INSTANT: ['Energy Usage', 'kW'],
```

Comment:
Not this one.  The API's getInstant call returns the value in W (which the component converts to kW), only getEnergy is returned in kWh: http://napi.hbcontent.com/document/index.php